src/components/transaction-graphic/MosaicSupplyRevocationGraphic.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+	<div>
+		<svg
+			version="1.1"
+			xmlns="http://www.w3.org/2000/svg"
+			xmlns:xlink="http://www.w3.org/1999/xlink"
+			x="0px"
+			y="0px"
+			:width="getPixels(transactionGraphicWidth)"
+			:height="getPixels(transactionGraphicHeight)"
+			:viewBox="transactionGraphicViewbox"
+			xml:space="preserve"
+		>
+			<AccountIcon
+				:x="subjectPositionX"
+				:y="subjectPositionY"
+				:width="subjectWidth"
+				:height="subjectHeight"
+				:address="address"
+			/>
+			<AccountIcon
+				:x="objectPositionX"
+				:y="objectPositionY"
+				:width="subjectWidth"
+				:height="subjectHeight"
+				:address="signer"
```

Comment:
Yes, I was intent to change because the Revocation transaction is the owner get back the mosaic.

Signer is getting mosaic from Target Address.

target Address ---mosaic icon----> Signer (current implementation)

what do you think about this?

Reply:
Yes, it makes sense